### PR TITLE
Make Jade stuff work correctly & logging

### PR DESCRIPTION
**Pull request by Floby**
Hello I corrected a few things that weren't working and added logging of each request. I'll try with socket.io and chrome

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ var app = module.exports = express.createServer();
 app.configure(function(){
   app.set('views', __dirname + '/views');
   app.set('view engine', 'jade');
+  app.use(express.logger({format: ":method :url :date from :remote-addr :user-agent -> :status\n"}));
```
Comment (Floby) [inline app.js:18]: this logs requests to the node server into the console.

**Comment by gnagno** [comment]
thank you I merged your work with mine, now I will try it as soon as I will have some time....

I was thinking about the frontend interface, do you think it will be better if the server will push the log lines enclosed in <div> or just followed by <br> ?

**Comment by Floby** [comment]
I set up socket.IO with your app. seems to work correctly on my computer. tested on chrome 10 (websocket) and firefox 3.6 (xhr-multipart). At the moment they just say hello to each other. I'll file your another pull request =)

**Comment by gnagno** [comment]
lol, I noticed now my previous message, it should be:
I was thinking about the frontend interface, do you think it will be better if the server will push the log lines enclosed in div tags or just followed by br tag ?

**Comment by gnagno** [comment]
Thank you for your work :)

my problem with google chrome was raising when I was watching the log from one file and at the sime time I was opening the same file with firefox, in result I was getting the same line more times

**Comment by Floby** [comment]
I remember, but I don't think it was due to socket.IO/chrome/firefox but more by wrong server logic.

**Comment by gnagno** [comment]
did you even try to follow different files from different browser tabs? I was getting an issue even in this case because every tab was receiving everything....

**Pull request merged**